addons/hr_holidays/models/hr_employee.py

```diff
@@ -281,6 +281,49 @@ def get_stress_days(self, start_date, end_date):
 
         return all_days
 
+    @api.model
+    def get_stress_days_data(self, date_start, date_end):
+        self = self or self.env.user.employee_id
+        stress_days = self._get_stress_days(date_start, date_end).sorted('start_date')
+        return list(map(lambda sd: {
+            'id': -sd.id,
+            'colorIndex': sd.color,
+            'end': datetime.datetime.combine(sd.end_date, datetime.datetime.max.time()).isoformat(),
+            'endType': "datetime",
+            'isAllDay': True,
+            'start': datetime.datetime.combine(sd.start_date, datetime.datetime.min.time()).isoformat(),
+            'startType': "datetime",
+            'title': sd.name,
+        }, stress_days))
+
+    @api.model
+    def get_bank_holidays_data(self, date_start, date_end):
+        self = self or self.env.user.employee_id
+        bank_holidays = self.env['resource.calendar.leaves'].search([
+            ('resource_id', '=', False),
+            ('company_id', 'in', self.company_id.ids + [False]),
```

Comment:
```suggestion
            ('company_id', 'in', (self.company_id.id, False)),
```
same for `calendar_id`